.github/ISSUE_TEMPLATE/05_type_pyaloads_incorrect_or_incomplete.md

```diff
@@ -0,0 +1,20 @@
+---
+name: "🐛 Payload Types Bug Report"
+about: "If a Payload Type is incorrect or incomplete according to Github's Documentation"
+labels: bug, typescript
+---
+
+<!-- Please replace all placeholders such as this below -->
+
+**Which Payload is failing/incomplete?**
+
+<!-- Please share which payload/s is/are failing -->
+
+**Example of a Payload fixed**
+
+<!-- Share a snippet proposal of a working payload example if possible -->
+<!-- you can get examples in https://github.com/octokit/webhooks/tree/master/payload-examples/api.github.com -->
+
+**Linked issue in @octokit/webhooks**
+
+<!-- Create a linked issue to this issue on @octokit/webhooks, once is fixed this package will get the fix in next update -->
```

Comment:
Not sure completely about the wording, it doesn't ring right to me. I'm not entirely sure how to rephrase it, but this is what I think is better.
Also added a link to the new issue page
```suggestion
<!-- Create an issue on @octokit/webhooks (https://github.com/octokit/webhooks/issues/new) and add a link back to this one. Once it is fixed there, this package will get the fix during the next update -->
```